src/packagedcode/rubygems.py

```diff
@@ -647,93 +648,124 @@ def normalize(gem_data, known_fields=known_fields):
     )
 
 
-def parse_spec(location):
-    pass
+# defaults fields of .gemspec file
+data_dic = {
```

Comment:
Why is this a global? 

Also, this needs a better name like `gemspec_data` or similar